src/lizaalert/quizzes/views.py

```diff
@@ -60,3 +81,65 @@ def create(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+class QuizDetailAnswerView(generics.CreateAPIView, generics.RetrieveAPIView):
+    """
+    Создает и обновляет запись UserAnswer для ответов пользователя на квиз.
+
+    При GET-запросе возвращает информацию о текущем UserAnswer пользователя для данного квиза.
+
+    При POST-запросе обновляет ответы пользователя и вычисляет результаты теста.
+    """
+
+    serializer_class = UserAnswerSerializer
+
+    def get_object(self):
+        user = self.request.user
+        lesson_id = self.kwargs["lesson_id"]
```

Comment:
```suggestion
        lesson_id = self.kwargs.get("lesson_id")
```

Если, по каким то причинам в kwargs не будет lesson_id, то все упадет с KeyError